documents/write_acls.go

```diff
@@ -22,15 +22,15 @@ type ChangedField struct {
 // GetChangedFields takes two document trees and returns the compact property, old and new value of the fields that are changed in new tree.
 // Properties may have been added to the new tree or removed from the new tree.
 // In Either case, since the new tree is different from old, that is considered a change.
-func GetChangedFields(oldTree, newTree *proofs.DocumentTree, lengthSuffix string) (changedFields []ChangedField) {
+func GetChangedFields(oldTree, newTree *proofs.DocumentTree) (changedFields []ChangedField) {
```

Comment:
this will make the function unusable as a generic call. Lets bring back the suffix, let caller pass the suffix